MetaMask/metamask-extension
# fix: Don't show AccountListMenu back button by default

## **Description**

A recent change (https://github.com/MetaMask/metamask-extension/pull/26271) made it such that the "<" (back button) in the `AccountListMenu` component always shows.  We do *not* want this when the `AccountListMenu` first opens, so this PR ensures that.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/26940?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Open the `AccountListMenu`
2. Don't see the `<`
3. Click the "Add account" button
4. See the `<`
5. Click the `<`, go back to main menu

## **Screenshots/Recordings**



### **Before**

<img width="891" alt="SCR-20240904-pens" src="https://github.com/user-attachments/assets/ee4b9cdc-503d-4d97-99a0-f7045ba1d503">


### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
